tutorials/openstack_workload_sfc.md

```diff
@@ -0,0 +1,201 @@
+ 	"virtualizationVersion": "Icehouse",
+	"softwareVersion": "0.3.0.5000",
+	"imageName": "cirrosWithTcpDump.qcow2",
+	"minIscVersion": {
+		"major": 0,
+		"minor": 0,
+		"build": 1,
+		"versionStr": ""
+	},
+	"minCpus": 2,
+	"memoryInMb": 1024,
+	"diskSizeInGb": 2,
+	"encapsulationTypes": ["VLAN"],
+	"additionalNicForInspection": false
+}
+```
+
+### SDN Controller  
+
+OSC requires two components to implement traffic redirection and SDN notifications through an SDN controller, an SDN component and an SDN controller plugin.
+
+For this tutorial, it is assumed that the SDN component and SDN controller plugin will be manually created:  
+ 
+* The [`SDN CONTROLLER NSFC PLUGIN`](/plugins/plugins.md) is uploaded on OSC, enabling communication between the SDN controller and OSC.
+* The **SDN Component** deployed on OpenStack is Neutron Service Function Chain
+
+
+## Set up OSC to Protect a Workload
+
+### 1. Upload Plugin
+
+Within OSC, navigate to **Manage** > **Plugins** using the left-hand menu. 
+* Upload the `SDN CONTROLLER NSFC PLUGIN`.  
+* Upload the `SAMPLE MANAGER PLUGIN`.  
+
+![Upload Plugins](images/add_plugins.jpg)  
+*Upload SDN Controller and Manager Plugins*
+
+### 2. Define Virtualization Connector  
+
+Using the left-hand menu, navigate to **Setup** > **Virtualization Connectors**, and then select **Add**.
+* Enter a name and then select **OPENSTACK** as the type.
+* Select **NSFC** (Network Service Function Chain Controller) as the type for the SDN Controller.
+* For the Keystone:
+	* Give the IP address of the `TENANT` environment in OpenStack.
+	* Enter `admin` as the Admin Tenant Name.
+	* Enter the OpenStack administrator credentials.
+
+![Add Virtualization Connector](images/add_VC_neutron_SFC.jpg)  
+*Add Virtualization Connector*
+
+### 3. [Define Manager Connector](openstack_workload.md#3-define-manager-connector) 
+### 4. [Define Service Function](openstack_workload.md#4-define-service-function) 
+### 5. [Define Distributed Appliance](openstack_workload.md#5-define-distributed-appliance) 
+### 6. [Define Deployment Specification](openstack_workload.md#6-define-deployment-specification) 
+### 7. [Define Security Group](openstack_workload.md#7-define-security-group) 
+
+### 8. Define a Service Function Chain
+At present we can create a service function only through REST APIs, there is no support in OSC UI for defining SFC.
+The OSC REST APIs provides all features as provided by OSC UI.
+We can access OSC REST APIs through WEB interface with URL as https://<OSC-IP>:8090/api-doc/
+
+Create a SFC as below
+* Click on `Operations for Virtualization Connector` link
+* Select  Parameter Content Type as `application/json`
+* Enter the value for VirtualizationConnector Id 
+* Enter the json body as below. Use model schema at right side as reference 
+
+![Create Service Fucntion Chain](images/SFC_Post_API.jpg)  
+*Create Service Fucntion Chain* 
+
+* To get the virtual system ids use get API call from Distributed Appliance API

```

Comment:
add  a note mentioning for a chain multiple vs ids can be provided in the order required.

Reply:
updated in chain with multiple sf section